DOCSP-32269 atlas top 250 revamp connection page (#458)
mongodb/docs-java: source/fundamentals/connection/connect.txt

Connect to MongoDB
==================

> This guide shows you how to connect to a 

Reviewer: Nitpick/Suggestion: I think the phrase "This guide shows you" is personifying the page. It seems more clear to me to stick with the "you can learn" format. (This is probably pretty subjective so feel free to take or leave this comment)

```suggestion
In this guide, you can learn how to connect to a 
```

connect to MongoDB and how it should behave while connected. The following
figure explains each part of a sample connection URI:

.. figure:: /includes/figures/dns_seedlist_connection_string_parts.png

>    :alt: Each part of the connection string

Reviewer 2: We should use better alt text here for accessibility. It should be meaningful to someone who can't see this image. Something like:
```suggestion
   :alt: An example of a connection string that demonstrates the protocol, credentials, hostname or IP, port, and connection options
```

> In this example, we connect to an Atlas MongoDB deployment that has a

Reviewer 2: Style: Write to the user by using second person. ([ref](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/))

```suggestion
In this example, you connect to an Atlas MongoDB deployment that has a
```

connect to MongoDB and how it should behave while connected. The following
figure explains each part of a sample connection URI:

.. figure:: /includes/figures/dns_seedlist_connection_string_parts.png
   :alt: Each part of the connection string

In this example, we connect to an Atlas MongoDB deployment that has a
DNS SRV record. For more details, see the :manual:`DNS Seed List
Connection Format
</reference/connection-string/#dns-seed-list-connection-format>`
documentation. This format offers flexibility in deployment and the
ability to change the servers in rotation without reconfiguring clients.

This format offers flexibility in deployment and the ability to change

> the servers in rotation without reconfiguring clients.

Reviewer: Repeated line

   and select Java from the language dropdown to retrieve your connection
   string.

If you are connecting to an instance or replica set that does not have a 
DNS SRV address, you must use ``mongodb`` for the protocol, which specifies 
the :manual:`Standard Connection String Format 
</reference/connection-string/#std-label-connections-standard-connection-string-format>`.

> After the protocol, the next part of the connection string contains

Reviewer: Suggestion: I think "after the protocol" and "the next part" mean the same thing. I think it could be simplified to:

```suggestion
After the protocol, the connection string contains your
```
